### PR TITLE
Multi qudit device

### PR DESCRIPTION
**Pull request by TheNewYorkerFlute**

### CODE REVIEW

**Review by fretchen — CHANGES_REQUESTED** [review]
recover all the changes to the examples.

File: pennylane_ls/MultiQuditDevice.py [hunk 1]
```diff
@@ -3,23 +3,17 @@
 import scipy
 
 from pennylane import Device
+from pennylane import QubitDevice
```
Comment (fretchen) [inline pennylane_ls/MultiQuditDevice.py:6]: really needed ?

File: pennylane_ls/MultiQuditDevice.py [hunk 2]
```diff
@@ -28,10 +22,29 @@ class MultiQuditDevice(Device):
 
     short_name = "synqs.sqs"
 
+    observables = {"Lz"}
+    operations  = {"rLz", "rLx", "RX", "CNOT", "ZZ", "XY"}
```
Comment (fretchen) [inline pennylane_ls/MultiQuditDevice.py:26]: just use the property function

File: pennylane_ls/MultiQuditDevice.py [hunk 3]
```diff
@@ -49,8 +62,9 @@ def capabilities(cls):
             model="qudit",
             supports_finite_shots=True,
             supports_tensor_observables=True,
-            returns_probs=True,
```
Comment (fretchen) [inline pennylane_ls/MultiQuditDevice.py:52]: revert this comment

File: pennylane_ls/MultiQuditDevice.py (hunk 1 above)
Comment (TheNewYorkerFlute) [reply pennylane_ls/MultiQuditDevice.py:6]: not needed

Comment (TheNewYorkerFlute) [reply pennylane_ls/MultiQuditDevice.py:6]: Done!

File: pennylane_ls/MultiQuditDevice.py (hunk 2 above)
Comment (TheNewYorkerFlute) [reply pennylane_ls/MultiQuditDevice.py:26]: Done!

Comment (TheNewYorkerFlute) [reply pennylane_ls/MultiQuditDevice.py:26]: Done!

File: pennylane_ls/MultiQuditDevice.py (hunk 3 above)
Comment (TheNewYorkerFlute) [reply pennylane_ls/MultiQuditDevice.py:52]: Done!

**Review by fretchen — APPROVED** [review]

**Pull request merged**